pkgs/applications/science/math/qalculate-gtk/default.nix

```diff
@@ -12,7 +12,11 @@ stdenv.mkDerivation rec {
   };
 
   patchPhase = ''
+    # The gnome-search-provider2.c file generated by gdbus-codegen depends
+    # on gio-unix-2.0, which is a Requires.private of gtk+-3.0,
+    # and private dependencies are dropped in our default patched pkg-config.
     # https://github.com/Qalculate/qalculate-gtk/pull/178
+    # https://github.com/NixOS/nixpkgs/issues/292
     substituteInPlace configure.ac --replace 'libxml-2.0' 'libxml-2.0 gio-unix-2.0'
```

Comment:
Looks like we can remove it in future: https://github.com/Qalculate/qalculate-gtk/commit/76a29d5f6d6cd18c11467a46dde4efee19c94ec6